tcl/file.tcl

```diff
@@ -124,6 +124,24 @@ proc ::file::openBaseAsTree { { fName "" } } {
   return $err
 }
 
+# switch to database when loaded or open the database
```

Comment:
This does not seem correct.
It should use ::file::SwitchToBase instead of `sc_base switch` and invoke ::notify::DatabaseChanged and ::notify::GameChanged